dbt_project.yml

```diff
@@ -6,6 +6,7 @@ require-dbt-version: [">=1.3.0", "<2.0.0"]
 vars:
   reddit_ads:
     account: "{{ source('reddit_ads', 'account') }}"
+    business_account: "{{ source('reddit_ads','business_account') }}"
```

Comment:
Interestingly it doesn't look like either of these variables ultimately are ever used. However, for consistency and to retain the scope of these changes, no action/change required but found this interesting. 

https://github.com/fivetran/dbt_reddit_ads/blob/b8bb39ba17df7a93c240dd6e436bae881a6e0d1e/models/staging/tmp/stg_reddit_ads__account_tmp.sql#L3-L8